homestar-runtime/src/event_handler/notification/network.rs

```diff
@@ -0,0 +1,2671 @@
+// Notification types for [swarm] events.
```

Comment:
Maybe `network_event` vs network?

Reply:
As discussed, this seems alright given the context of it being in the notifications directory.